modules/juce_audio_devices/native/juce_ios_Audio.cpp

```diff
@@ -1034,16 +1034,12 @@ struct iOSAudioIODevice::Pimpl      : public AsyncUpdater
         setAnalogInputGain(0.5f);
         gainCompensation = 1.f;
         
-        if (!AudioIODeviceType::useDeviceVoiceProcessing)
+        if (isUsingBuiltInSpeaker() && !AudioIODeviceType::useDeviceVoiceProcessing)
         {
             setAudioPreprocessingEnabled(false);

```

Comment:
Do we need to set this to true in an else block?

Reply:
I think it's true by default since my bluetooth headphones are working now with this code. I can add it before just to be safe